### ISSUE TITLE
Clean up plan-all output

### ISSUE THREAD

**Issue by andrewhertog** [gruntwork-io/terragrunt#431]
Right now when you run a plan all on a directory with multiple files to plan, the output can be quite unorganized, and hard to parse through, especially when there are errors. 

From my understanding, plan-all runs a plan on all the available files in parallel, and then spits out the output as it is available from plan. Several plans can finish at the same time and spit out its output all at the same time. Maybe buffering all the output until all the plans are complete so that no two plans are writing their output at the same time?

**Comment by brikis98** [comment]
Agreed. PR to improve that would be very welcome. See also #74.
